apps/src/templates/FinishDialog.story.jsx

```diff
@@ -0,0 +1,96 @@
+import { UnconnectedFinishDialog as FinishDialog } from './FinishDialog';
+import ExampleDialogButton from '../util/ExampleDialogButton';
+import React from 'react';
+
+const dialogWrapper = {
+  padding: 40,
+  width: 375,
+  backgroundColor: 'gray',
+};
+
+const achievements = [
+  {
+    isAchieved: true,
+    iconUrl: '',
+    message: 'No hints used!',
+  },
+  {
+    isAchieved: true,
+    iconUrl: '',
+    message: 'Achievement got',
+  },
+  {
+    isAchieved: false,
+    iconUrl: '',
+    message: 'Achievement missed',
+  },
+];
+
+export default storybook =>
+  storybook
+    .storiesOf('FinishDialog')
+    .addStoryTable([
+      {
+        name: 'Dialog with overlay',
+        description: 'Perfectly finished, no share button, thumbnail, achievements, etc.',
+        story: () => (
+          <ExampleDialogButton>
```

Comment:
Nit: can we connect up the buttons to close `ExampleDialogButton`?  Otherwise the only way to close it is to refresh the page.